### ISSUE TITLE
Feature request: !issues command

### ISSUE THREAD

**Issue by mmahut** [tulak/zabbirc#2]
Would be nice to have an !issues command that would list top 10 active unacknowledged triggers sorted by priority (and probably time unacked).

```
<marek> !issues
<zabbirc> Top 10 issues:
<zabbirc> |1404337765| |[high] System down X
<zabbirc> |1404337785| [aveage] High I/O Wait on node X
...
<marek> !ack 1404337765 Working on this top issue right now
...
```
